libs/features/personalization/manifest-utils.js

```diff
@@ -19,15 +20,46 @@ export const normalizePath = (p) => {
   return path;
 };
 
+const GMTStringToLocalDate = (gmtString) => new Date(`${gmtString}+00:00`);
+const getEvents = () => {
+  const events = getMetadata('events');
+  return events ? events.split(',').map((event) => {
+    const [name, start, end] = event.trim().split('|').map((s) => s.trim());
+    return { name, start: GMTStringToLocalDate(start), end: GMTStringToLocalDate(end) };
+  }) : [];
+};
+
+const getEventInfo = (manifestPath, events) => {
+  let disabled = false;
+  let event = null;
+  const scheduled = manifestPath.match(REGEX_PROMO);
+  if (scheduled) {
+    event = events.find((e) => e.name === scheduled[1]);
+    if (!event) {
+      disabled = true;
+    } else {
+      const currentDate = new Date();
+      if (event.start && event.end
```

Comment:
don't we want to manage "evergreen" promotion that are endless?

Reply:
for now, i don't recall such use-case.
can you give more details?
how will it work?

Reply:
never mind those use case are not meant to leverage mep anyway.

Reply:
> don't we want to manage "evergreen" promotion that are endless?

Evergreen sounds like regular content then, no? Should not be tied to promotion anymore in that case.